gymnasium_robotics/utils/mujoco_utils.py

```diff
@@ -192,7 +192,7 @@ def get_joint_qpos(model, data, name):
     start_idx = joint_addr
     end_idx = joint_addr + ndim
 
-    return data.qpos[start_idx:end_idx]
+    return np.array(data.qpos[start_idx:end_idx])
```

Comment:
Why not use `.copy()`, this is the recommended method on  `numpy` DOCS (https://numpy.org/doc/stable/reference/generated/numpy.ndarray.copy.html#numpy.ndarray.copy)


Does this even work? dont you need to do `np.array(my_old_array, copy=True) `

Reply:
Changed to use `np.copy`. 

Yes, calling the `np.array` without `copy` argument works because it is set to true by default.

Reply:
i meant to use `numpy.ndarray.copy` and not `numpy.copy`

(https://numpy.org/doc/stable/reference/generated/numpy.ndarray.copy.html#numpy.ndarray.copy))

Reply:
Fixed.